package.json

```diff
@@ -88,7 +89,7 @@
     "eslint-config-next": "^13.4.3",
     "eslint-plugin-storybook": "^0.6.12",
     "husky": "^8.0.3",
-    "linkfree-cli": "^2.3.1",
+    "linkfree-cli": "^2.2.2",
```

Comment:
I think this is due to a bad merge from `main`?

Reply:
I didn't change that. I've seen that you merged main into this branch, so maybe that's why?

Reply:
I merged main to try and fix it, but it seems your branch has the old version